src/reactviews/pages/ConnectionDialog/connectionHeader.tsx

```diff
@@ -27,7 +26,7 @@ export const ConnectionHeader = () => {
 					padding: '10px',
 				}
 			}
-				src={connectionDialogContext?.theme === webLightTheme ? loadImage(sqlServerImage) : loadImage(sqlServerImageDark)} alt='SQL Server' height={60} width={60} />
+				src={connectionDialogContext?.theme === webLightTheme ? sqlServerImage : sqlServerImageDark} alt='SQL Server' height={60} width={60} />

```

Comment:
this alt-text probably should be localized

Reply:
Well fix this in my webview localization PR